spec/elastic_apm/config_spec.rb

```diff
@@ -42,7 +42,7 @@ module ElasticAPM
         ['ELASTIC_APM_VERIFY_SERVER_CERT', 'true', true],
         ['ELASTIC_APM_VERIFY_SERVER_CERT', '0', false],
         ['ELASTIC_APM_VERIFY_SERVER_CERT', 'false', false],
-        ['ELASTIC_APM_DISABLED_SPIES', 'json,http', %w[json http]],
+        ['ELASTIC_APM_DISABLED_INSTRUMENTATIONS', 'json,http', %w[json http]],
```

Comment:
And here wouldn't we want to keep the test for `ELASTIC_APM_DISABLED_SPIES` as the option hasn't been fully removed yet?

Reply:
Added a dedicated example in the deprecations section